test/SFSmartSyncTestSuite.js

```diff
@@ -551,15 +551,11 @@ SmartSyncTestSuite.prototype.testStoreCacheWithGlobalStoreNamed = function() {
     var agent008 = {Id:"008", Name:"Vilain"};
     var querySpec007 = {queryType:"exact", indexPath:"Name", matchKey:"JamesBond", order:"ascending", pageSize:1}
     var querySpec008 = {queryType:"exact", indexPath:"Name", matchKey:"Vilain", order:"ascending", pageSize:1}
-    var storeConfigWithName =  {"isGlobalStore" : false,'storeName' : 'NAMED_USR_STORE'};
-    var globalStoreConfigWithName =  {"isGlobalStore" : true,'storeName' : 'NAMED_GLBL_STORE'};
+    var storeConfigWithName =  {"isGlobalStore" : false, "storeName" : "userStoreFor_" + this.module.currentTestName};
+    var globalStoreConfigWithName =  {"isGlobalStore" : true, "storeName" : "globalStoreFor_" + this.module.currentTestName};
 
-    Promise.all([Force.smartstoreClient.removeAllStores(),
```

Comment:
I suspect removeAllStores is the reason for the intermittent failures of that test.